src/views/virtualmachines/list/hooks/sortColumns.ts

```diff
@@ -53,12 +53,12 @@ export const sortByNetworkUsage = (
   pagination: { [key: string]: any },
 ) => {
   const compareCPUUsage = (a: V1VirtualMachine, b: V1VirtualMachine): number => {
```

Comment:
Compare CPU usage is comparing class names?

Reply:
fixed